src/PdfReader/index.tsx

```diff
@@ -4,14 +4,15 @@ import {
   ColorMode,
   ReaderArguments,
   ReaderReturn,
-  ReaderState,
   WebpubManifest,
+  PdfReaderState,
 } from '../types';
 import { chakra, Flex, shouldForwardProp } from '@chakra-ui/react';
+import useContainerWidth from '../ui/hooks/useContainerWidth';
```

Comment:
Did you add this import? It looks like it isn't used

Reply:
Removed! I am not sure why I imported that..